ansible/roles/linux-zenith-client/templates/zenith-playbook.yml

```diff
@@ -21,7 +21,7 @@
     - name: Set Zenith facts
       set_fact:
         zenith_client_registrar_url: "{{ os_user_metadata.zenith_registrar_url }}"
-        zenith_client_registrar_token: "{{ os_user_metadata.zenith_registrar_token }}"
+        zenith_client_registrar_token: "{{ os_user_metadata.{% endraw %}{{ zenith_registrar_token_metadata_key }}{% raw %} }}"
```

Comment:
There is an easier/cleaner way to do this. This becomes:

```yaml
zenith_client_registrar_token: "{{ os_user_metadata[zenith_registrar_token_metadata_key] }}"
```

then write the metadata key to the vars file for the client.

Reply:
This is a much better way to do this - updated.